integration/kubernetes/k8s-footloose.bats

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+load "${BATS_TEST_DIRNAME}/../../lib/common.bash"
+
+setup() {
+	export KUBECONFIG="$HOME/.kube/config"
+	pod_name="footfed"
+	config_name="ssh-pub-key"
```

Comment:
I would suggest to rename as `configmap_name="ssh-config-map"` to be more specific about the purpose of the variable